src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./blocks/body";
+@import "./blocks/card";
+@import "./blocks/stars";
```

Comment:
where is your styles, I didn't get it? maybe you forgot to stash them before commiting?

Reply:
I apologize, I have added all the files